### DISCUSSION TITLE
Cancel button should be capitalized

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#23431] [New to GitHub]
<p>Go to any page in the forum and try to create/reply:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/6/f640e3c25d8080cbd2dfcf047d8ae0293ee0216f.png" data-download-href="/uploads/short-url/z8stXzeOJKirvfU6WrpmNa2w6lx.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2119212/181076888-5112e558-f627-4ad9-b862-1492dd24f865.png" alt="image" data-base62-sha1="z8stXzeOJKirvfU6WrpmNa2w6lx" width="690" height="363" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f640e3c25d8080cbd2dfcf047d8ae0293ee0216f_2_690x363.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f640e3c25d8080cbd2dfcf047d8ae0293ee0216f_2_1035x544.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f640e3c25d8080cbd2dfcf047d8ae0293ee0216f_2_1380x726.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f640e3c25d8080cbd2dfcf047d8ae0293ee0216f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1448×762 39.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Note that the <code>Create Topic</code> button is capitalized (or <code>Reply</code> as applicable) and the <code>Cancel</code> is not.</p>
<p>Expected results:</p>
<p><code>Cancel</code> should be capitalized (Title Case apparently)</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/jsoref">@jsoref</a>!</p>
<p>Thank you for sharing your suggestion and feedback on the button design.  I’ll relay this to our team here.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>We’ve updated the title case.  Thank you for taking the time to highlight that. :slightly_smiling_face:</p>
